### DISCUSSION TITLE
Control Panel

### DISCUSSION THREAD

**Discussion by studiosteve64** [jgyates/genmon#1132] [General]
I have a R panel, 2008 model liquid cooled, curious if this model would be adaptable to your WiFi solution?
![image0](https://github.com/jgyates/genmon/assets/171492957/6a02e537-59c8-4ac0-adca-4f929ccf5a7f)
![R control](https://github.com/jgyates/genmon/assets/171492957/7ac434aa-f823-40bb-8c84-1a571b3c4e74)
![R-control1](https://github.com/jgyates/genmon/assets/171492957/e67b8305-4122-4bf1-a4f9-571f7c53fbfa)

**Comment by jgyates** [comment]
Unfortunately no. This generator does not support modbus serial data. this is how genmon reads the generator status so this generator is not supported by this project.

**Reply by studiosteve64** [reply]
Have any ideas on how a person might do some monitoring with this control panel?

**Reply by jgyates** [reply]
I don't know a lot about that controller, but you could possibly use an embedded computer, like a raspberry pi to monitor GPIO lines connected to the LEDs on the controller. 

You could also opt for a third party solution like this: https://www.flightsystems.com/genpulse
